components/automate-ui/src/app/modules/license/license-notifications/license-notifications.component.html

```diff
@@ -1,13 +1,14 @@
 <ng-container *ngFor="let notification of notifications">
-  <chef-notification
+  <app-chef-notification
```

Comment:
I think it is more consistent to use this:
```suggestion
  <app-notification
```
... because the wrapper is `<app-notifications>`, not `<app-chef-notifications>`